src/integration-test/java/se/sundsvall/notes/apptest/CreateNoteIT.java

```diff
@@ -24,7 +23,6 @@
  * Create note apptests.
  */
 @WireMockAppTestSuite(files = "classpath:/CreateNoteIT/", classes = Application.class)
-@ActiveProfiles("junit")
 class CreateNoteIT extends AbstractAppTest {
```

Comment:
I generally like the idea of letting app-tests perform "black-box-testing" (instead of analyzing the internals)

But I have added verifications for this now.